### PR TITLE
chore(parse-i3s-tile-content): TS, refactor.

### PR DESCRIPTION
**Pull request by HeimEndyd**

### CODE REVIEW

**Review by belom88 — APPROVED** [review]

File: modules/i3s/src/lib/helpers/i3s-nodepages-tiles.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ export default class I3SNodePagesTiles {
     let contentUrl = null;
     let textureUrl: string | null = null;
     let materialDefinition = null;
-    let textureFormat = 'jpeg';
+    let textureFormat = 'jpg';
```
Comment (ibgreen) [inline modules/i3s/src/lib/helpers/i3s-nodepages-tiles.ts:80]: Why the change? Seems unrelated?

File: modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts [hunk 2]
```diff
@@ -6,54 +6,53 @@ import {Ellipsoid} from '@math.gl/geospatial';
...
   }
 }
 
 const I3S_ATTRIBUTE_TYPE = 'i3s-attribute-type';
 
 export async function parseI3STileContent(
   arrayBuffer: ArrayBuffer,
-  tile: Tile,
-  tileset: Tileset,
+  tile: I3sTileHeader,
```
Comment (ibgreen) [inline modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts:54]: `I3S` capitals...

File: modules/i3s/src/types.ts [hunk 3]
```diff
@@ -18,35 +19,37 @@ export enum DATA_TYPE {
  * spec - https://github.com/Esri/i3s-spec/blob/master/docs/1.8/3DSceneLayer.cmn.md
  */
 // TODO Replace "[key: string]: any" with actual defenition
-export type Tileset = {
+export type I3sTilesetHeader = {
```
Comment (ibgreen) [inline modules/i3s/src/types.ts:22]: capitals
```suggestion
export type I3STilesetHeader = {
```

**Review by ibgreen — APPROVED** [review]

File: modules/i3s/src/lib/helpers/i3s-nodepages-tiles.ts (hunk 1 above)
Comment (HeimEndyd) [reply modules/i3s/src/lib/helpers/i3s-nodepages-tiles.ts:80]: In spec - https://github.com/Esri/i3s-spec/blob/master/docs/1.8/textureSetDefinitionFormat.cmn.md no "jpeg" texture format, only "jpg". Because of this, in the getLoaderForTextureFormat function, instead of choosing a loader, the default loader was selected.
